addons/hr_holidays/security/ir_rule.xml

```diff
@@ -4,7 +4,7 @@
     <record id="property_rule_holidays_employee" model="ir.rule">
         <field name="name">Employee Holidays</field>
         <field name="model_id" ref="model_hr_holidays"/>
-        <field name="domain_force">[('employee_id.user_id','=',user.id)]</field>
+        <field name="domain_force">['|', ('employee_id.user_id','=',user.id), ('state', '=', 'validate')]</field>
```

Comment:
@dmo-odoo @tde-banana-odoo @KangOl 

Any valuable reason to modify the ir.rule ? Seems strange to give access to the record only to display a label. (With @mart-e and @msh-odoo )